### ISSUE TITLE
How can I embed two or more PApplets?

### ISSUE THREAD

**Issue by ylascer** [micycle1/ProcessingFX#8]
According to your example, it starts from one PApplet, and then launce JavaFX GUI.
so then, if I want two or more PApplets, what should I do?

And another question is~, 
Is there any way to start JavaFX GUI at first and then call(embed) the PApplets as componet of GUI ?

It'll be very thankful if you provide an example code.

**Comment by micycle1** [comment]
Do you mean 2 PApplets within one JavaFX stage?

**Comment by ylascer** [comment]
> Do you mean 2 PApplets within one JavaFX stage?

Yep. in one JavaFX stage, and maybe think about divided two or more anchorpanes, for example 1 PApplet in one anchorpane, and anothor PApplet in anothor anchorpane. 
Is it possible ??
====< Add more information for my intention>===
Basically, I want to execute Java programe, with JavaFX GUI.
there may be many other components in GUI.
finally, two or more PApplets embeded in two or more anchorpanes each. and PApplets interacts with other JavaFX components.  
PApplet's anchorpane (exactly javafx canvas) could be resizable, maximizable, minimizable without influence.

**Comment by micycle1** [comment]
You could perhaps try having 2 `App.surface` variables and assign the surface of a PApplet to each, then launch the JavaFX app.
